styleguide/source/assets/css/scss/generic/_mixins.scss

```diff
@@ -26,6 +26,11 @@
   border-top: $width $color $style;
 }
 
+// Horizontal rule
+@mixin rule-horizontal-bottom($width, $color, $style) {
```

Comment:
I think the mixin approach is straightforward IMO so I'm going to use it in https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/162